.pylintrc

```diff
@@ -0,0 +1,16 @@
+[MAIN]
+ignore-paths=.*/migrations 
+
+disable=
+    line-too-long,
```

Comment:
I don't think that one should be disabled. Formatters will almost always format to the correct length and it's a good reminder to run them.